examples/src/main/java/org/apache/spark/examples/ml/JavaQuantileDiscretizerExample.java

```diff
@@ -58,7 +58,11 @@ public static void main(String[] args) {
     QuantileDiscretizer discretizer = new QuantileDiscretizer()
       .setInputCol("hour")
       .setOutputCol("result")
-      .setNumBuckets(3);
+      .setNumBuckets(3)
+      .setRelativeError(0);
+      // Note that we compute exact quantiles here by setting `relativeError` to 0 for
```

Comment:
I actually think it will be better to put the comment above the code block since it spans two lines (for each example)